### ISSUE TITLE
Design & Implement Clustered CacheManager Creation CLI

### ISSUE THREAD

**Issue by chrisdennis** [ehcache/ehcache3#738]
We need to design a CLI for creating a clustered cache manager that could be used by an ops/admin.  This could either be an executable JAR to be used directly or a bash script that launches a Java process.  The second approach probably fits better with existing approaches in Terracotta.

This is obviously dependent on #722 (and likely #723 as well).

**Comment by chrisdennis** [comment]
For now I am leaving this as a single task... I don't anticipate this being particularly complex.  We can split out additional tasks as necessary.

**Comment by chrisdennis** [comment]
jolt-simple looks like a nice solution for command-line parsing here... maybe we could adopt it uniformly even?

**Comment by ljacomet** [comment]
Heard good things of JCommander too

**Comment by chrisdennis** [comment]
I don't believe anyone currently think this is a good idea.  If it comes up again I'd rather have a new issue with a newly evaluated set of requirements.
